src/public/Interfaces.ts

```diff
@@ -155,6 +155,19 @@ export interface ExecutionContext {
     functionDirectory: string;
 }
 
+export interface TraceContext {
+    /** Populated during function invocation and corresponds to Activity.Current.Id during the invocation. */
+    Traceparent: string | null | undefined;
+
+    /** Populated during function invocation and corresponds to Activity.Current.TraceStateString during the invocation. */
+    Tracestate: string | null | undefined;
+  
+    /** Populated during function invocation and corresponds to Activity.Current.Tags during the invocation. */
+    Attributes: {
```

Comment:
(i'm assuming attributes and tracestate may not exist while traceparent always does, which is why those are optional with the question mark. is that a fair assumption?)
```javascript
attributes?: { 
    [k: string]: string
};
```